apps/bond/.eslintrc.json

```
@@ -0,0 +1,37 @@
+{
+  "extends": [
+    "plugin:@nx/react-typescript",
+    "next",
```

only need either next or next/core-web-vitals
core-web-vitals extends next with performance optimization suggestions, ie. using useCallback useMemo and more
```suggestion
    "next",
```